py3-clients-requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Zaza
 git+https://github.com/openstack-charmers/zaza.git#egg=zaza
+git+https://github.com/openstack-charmers/zaza-openstack-tests.git#egg=zaza-openstack-tests
```

Comment:
I am not sure if it matters what the egg is called, but we have in the charms.
git+https://github.com/openstack-charmers/zaza-openstack-tests.git#egg=zaza.openstack